flang/test/Lower/OpenMP/parallel-reduction-private.f90

```diff
@@ -0,0 +1,30 @@
+! Test that reductions and delayed privatization work properly togehter. Since
```

Comment:
Wouldn't delayed-privatization-reduction.f90 be a better name for this test?

Reply:
Indeed. Changed.